src/chrome/content/rules/Showcase.ca.xml

```diff
@@ -1,65 +1,11 @@
 <!--
-	For problematic rules, see Showcase.ca-problematic.xml.
-
-
 	Other Showcase Media rulesets:
-
 		- Smdg.ca.xml
-
-
-	CDN buckets:
-
-		- s3.amazonaws.com/smdgstatic/
-
-		- d2wk97pdwmv7mo.cloudfront.net
-
-			- media
-
-		- www.showcase.ca.edgesuite.net
-
-			- a1694.g.akamai.net
-
-
-	Problematic subdomains:
-
-		- ^ ¹
-		- media ²
-		- www ³
-
-	¹ Refused
-	² cloudfront
-	³ Works, akamai
-
-
-	Mixed content:
-
-		- Images, on www from:
-
-			- media ¹
-			- static.smdg.ca ¹
-
-		- favicon from $self ¹
-
-	¹ Secured by us
-
 -->
-<ruleset name="Showcase.ca (partial)">
-
+<ruleset name="Showcase.ca">
+	<target host="showcase.ca" />
+	<target host="www.showcase.ca" />
 	<target host="media.showcase.ca" />
```

Comment:
Also try:

```
assets.showcase.ca
uat.assets.showcase.ca
contests.showcase.ca
assets.contests.showcase.ca
uat.assets.contests.showcase.ca
l.showcase.ca
mail.showcase.ca
on.showcase.ca
preprod.showcase.ca
testsite.showcase.ca
tr.showcase.ca
```

Reply:
I just try to clear and close #15916 ……

Reply:
This should go into a follow-up PR then.